### PR TITLE
fix(files): init sidebar panels in injection context

### PR DESCRIPTION
**Pull request by dschmidt**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by JammingBen — APPROVED** [review]
Arghs good catch!

File: packages/web-app-files/src/extensions.ts [hunk 1]
```diff
@@ -19,10 +19,12 @@ export const extensions = ({ applicationConfig }: ApplicationSetupOptions) => {
   const { actions: showSharesActions } = useFileActionsShowShares()
   const { actions: quickLinkActions } = useFileActionsCopyQuickLink()
 
+  let panels = sideBarPanels()
```
Comment (kulmann) [inline packages/web-app-files/src/extensions.ts:22]: Why not `const`?

Comment (dschmidt) [reply packages/web-app-files/src/extensions.ts:22]: No reason, fixing...

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**